pypeit/specobj.py

```diff
@@ -117,7 +118,7 @@ class SpecObj(datamodel.DataContainer):
                                           descr='Boxcar extracted noise variance, sky+read noise '
                                                 'only (counts^2)'),
                  'BOX_MASK': dict(otype=np.ndarray, atype=np.bool_,
-                                  descr='Mask for optimally extracted flux'),
+                                  descr='Mask for optimally extracted flux. True=good'),
```

Comment:
Should this be "...boxcar extracted flux..."?

Reply:
good catch